README.rst

```diff
@@ -0,0 +1,69 @@
+Sentry Zendesk
+==============
+
+.. image:: https://img.shields.io/pypi/v/sentry-zendesk.svg
+    :target: https://pypi.python.org/pypi/sentry-zendesk
+
+.. image:: https://img.shields.io/pypi/pyversions/sentry-zendesk.svg
+    :target: https://pypi.python.org/pypi/sentry-zendesk
+
+.. image:: https://img.shields.io/pypi/l/sentry-zendesk.svg
+    :target: https://pypi.python.org/pypi/sentry-zendesk
+
+.. image:: https://travis-ci.org/ESSS/sentry-zendesk.svg?branch=master
+    :target: https://travis-ci.org/ESSS/sentry-zendesk
+    :alt: See Build Status on Travis CI
+
+.. image:: https://codecov.io/gh/ESSS/sentry-zendesk/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/ESSS/sentry-zendesk?branch=master
+   :alt: Coverage Status
+
+
+Plugin for Sentry which allows linking Zendesk tickets to Sentry issues
+
+    **DISCLAIMER**: Sentry API under development and `is not frozen <https://docs.sentry.io/server/plugins/>`_.
```

Comment:
This renders [quite ugly](https://github.com/ESSS/sentry-zendesk/tree/basic-features). I suggest removing the indentation.

Also consider moving this under **Limitations**.